### PR TITLE
Tealium

### PR DESCRIPTION
**Pull request by jeffreystutz**
# Pull Request/Issue Resolution

#### Is this change associated with a Braze feature/product release?
- [X ] Yes (**3/22/22**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the chart below to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then `Docs`, this can take up to 5 minutes to load. A `502` error means the deployment is still loading, refresh the page, and try again.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Application Infrastructure<br>Data Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Intelligence<br>Product Partnerships<br>SMS<br>Internal Tools</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Ingestion<br>Platforms and Channels<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Messaging and Automation<br>Dashboard Infrastructure<br>Email</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by jeffreystutz** [comment]
Feedback welcome; please do not merge yet.

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+> [Tealium](https://www.tealium.com) is a customer data platform that collects and routes information from multiple sources to a variety of other locations in your marketing stack.
+
+The Braze and Tealium integration allows you to seamlessly control the flow of information between the two systems. With Currents, you can also connect data to Tealium to make it actionable across the entire growth stack. 
+
+## Prerequisites
+
+| Requirement | Description |
+| ----------- | ----------- |
+| Tealium EventStream and/or Tealium AudienceStream | A [Tealium account](https://my.tealiumiq.com/) is required to take advantage of this partnership. |
+| Currents | In order to export data back into mParticle, you need to have [Braze Currents]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/#access-currents) set up for your account. |
```
Comment (alex-braze) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md:24]: This mentions "mParticle". We should update this to be "...to export data back to Tealium,.."

Comment (jeffreystutz) [reply _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md:24]: Oops thought I'd gotten all of those. Good catch, thank you!

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+### Step 1: Create a Data Source for Braze within Tealium
+
+Instructions for this step are documented [here](https://community.tealiumiq.com/t5/Customer-Data-Hub/Data-Sources/ta-p/17933#toc-hId--270626666). At the end, Tealium will provide a Data Source URL to copy, which you will use in the next step.
+
+### Step 2: Create Current
+
+In Braze, navigate to **Currents > + Create Current > Create Tealium Export**. Provide an integration name,  contact email and the Tealium URL from Step 1. Next, select what you want to track from the list of available events. Lastly, click **Launch Current**
+
+{% alert important %}
+It's important to keep your Tealium URL up to date; if your connector's credentials expire, the connector will stop sending events. If this persists for more than **48 hours**, the connector's events will be dropped and data will be permanently lost.
```
Comment (bm1549) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md:39]:
```suggestion
It is important to keep your Tealium URL up to date; if your connector's credentials expire, the connector will stop sending events. If this persists for more than **48 hours**, the connector's events will be dropped and data will be permanently lost.
```

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+| Tealium EventStream and/or Tealium AudienceStream | A [Tealium account](https://my.tealiumiq.com/) is required to take advantage of this partnership. |
+| Currents | In order to export data back into Tealium, you need to have [Braze Currents]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/#access-currents) set up for your account. |
+| Tealium URL | These can be obtained by navigating to your Tealium dashboard and copying the ingestion URL .|
+{: .reset-td-br-1 .reset-td-br-2}
+
+## Integration
+
+### Step 1: Create a Data Source for Braze within Tealium
+
+Instructions for this step are documented [here](https://community.tealiumiq.com/t5/Customer-Data-Hub/Data-Sources/ta-p/17933#toc-hId--270626666). At the end, Tealium will provide a Data Source URL to copy, which you will use in the next step.
```
Comment (darthryking) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md:32]: This should probably specify that the customer will need to specifically use the "Braze" platform for their data source, which isn't really specified anywhere in that Tealium documentation.

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+### Step 2: Create Current
+
+In Braze, navigate to **Currents > + Create Current > Create Tealium Export**. Provide an integration name,  contact email and the Tealium URL from Step 1. Next, select what you want to track from the list of available events. Lastly, click **Launch Current**
+
+{% alert important %}
+It's important to keep your Tealium URL up to date; if your connector's credentials expire, the connector will stop sending events. If this persists for more than **48 hours**, the connector's events will be dropped and data will be permanently lost.
+{% endalert %}
+
+All events sent to Tealium will include the user's `external_user_id`. At this time, Braze does not send event data for users who do not have their `external_user_id` set.
```
Comment (darthryking) [inline _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md:42]: Specifically Braze does not send event data *to Tealium* for users who do not have `external_user_id` set.

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md (hunk 2 above)
Comment (jeffreystutz) [reply _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md:39]: Any particular reason for removing the contraction?

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md (hunk 3 above)
Comment (jeffreystutz) [reply _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md:32]: Updated the link. [New one](https://community.tealiumiq.com/t5/Customer-Data-Hub/Braze-Currents-Incoming-Webhook-Setup-Guide/ta-p/36303) is more explicit. (Note: Working on getting the "webhook" language removed from their doc)

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md (hunk 4 above)
Comment (jeffreystutz) [reply _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md:42]: good point. Fixed.

File: _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md (hunk 2 above)
Comment (bm1549) [reply _docs/_partners/data_and_infrastructure_agility/customer_data_platform/tealium/tealium_for_currents.md]: Contractions are a part of informal writing. For external-facing documentation, we should keep things formal

**Review by KellieHawks — APPROVED** [review]

**Review by bm1549 — APPROVED** [review]

**Pull request merged**